contracts/Treasury.sol

```diff
@@ -43,9 +45,12 @@ contract OlympusTreasury is OlympusAccessControlled, ITreasury {
         LIQUIDITYTOKEN,
         LIQUIDITYMANAGER,
         RESERVEDEBTOR,
+        RISKRESERVETOKEN,
```

Comment:
`// Allows a RESERVETOKEN to use a custom riskOffValuation when calculating tokenValue`